packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -69,8 +68,12 @@ export class BlobManager {
         });
     }
 
+    private hasBlob(id: string): boolean {
+        return this.blobIds.has(id) || this.detachedBlobIds.has(id);
+    }
+
     public async getBlob(blobId: string): Promise<IFluidHandle<ArrayBufferLike>> {
-        assert(this.blobIds.has(blobId) || this.pendingBlobIds.has(blobId), 0x11f /* "requesting unknown blobs" */);
```

Comment:
What happened to the pendingBlobIds check here?

Reply:
This is the result of a previous change which made it impossible for blobs to be requested while pending (see `createBlob()`/`processBlobAttachOp()`)